index.js

```diff
@@ -11,6 +16,12 @@ module.exports = function vueify (file) {
   function dependency(file) {
     stream.emit('file', file)
   }
+
+  function style(buf) {
+    if (config.cssExport) {
+      fs.appendFile(config.cssExport, buf)
```

Comment:
would this result in appending to the file multiple times if the same files keeps being edited?